client/src/components/Profile.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { Row, Col, Input, Button } from 'react-materialize';
+import { server_url } from '../config.json';
+
+
+export default class Profile extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            Edit: false,
+            msg: '',
+            user: [],
+        };
+
+        this.UpdateClick = this.UpdateClick.bind(this);
+        this.onChangeHandler = this.onChangeHandler.bind(this);
+        this.handleClick_Main = this.handleClick_Main.bind(this);
+        this.handleClick_Edit = this.handleClick_Edit.bind(this);
+    }
+
+    componentDidMount() {
+        fetch(server_url + '/profile/' + window.location.href.split("/").pop())
```

Comment:
Why you don't use request function from utils.js?

Reply:
I wrote this piece of code before utils.js was implemented

Reply:
But now it's here, it takes less space and seems better. Also if we will need to upgrade it, better to do this in one place.

Reply:
Also I don't understand `window.location.href.split("/").pop()`, you define this route like ` <Route path="/profile/:profile_id" component={Profile}/>` so the path parameter stored in this.props.match.params.profile_id